apps/www/public/registry/themes.css

```diff
@@ -249,7 +249,7 @@
   --accent: 215 27.9% 16.9%;
   --accent-foreground: 210 20% 98%;
 
-  --destructive: 0 62.8% 30.6%;
+  --destructive: 0 72.8% 47%;
```

Comment:
Did you check if the red color is passing the contrast color  for accessibility? I saw your example and I didn't test it but I think the red should be brightner. You can use this website to verifiy if is passing or not.

https://webaim.org/resources/contrastchecker/